docs/docs/models/td-full-metrics.md

```diff
@@ -0,0 +1,162 @@
+# Time dependent model performance
+
+## Species model
+
+The model was trained using 249,269 videos 14 countries in West, Central, and East Africa.
+To evaluate the performance of the model, we held out 30,324 videos from 689 randomly-chosen transects.
+Importantly, all videos from each transect were either used for training or held out for evaluation.
+So the performance of the model on the holdout set should reflect its performance on videos from a transect
+the model has never seen.
+
+### Description of the holdout set
+
+All 14 countries are represented in the holdout set; the following table shows the number of videos from
+each country.
+These proportions are roughly consistent with the proportions in the training set.
+
+<table>
+  <thead>
+    <tr>
+      <th>Country</th>
+      <th>Number of videos</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>Ivory Coast</th>
+      <td>10987</td>
+    </tr>
+    <tr>
+      <th>Guinea</th>
+      <td>4300</td>
+    </tr>
+    <tr>
+      <th>DR Congo</th>
+      <td>2750</td>
+    </tr>
+    <tr>
+      <th>Uganda</th>
+      <td>2497</td>
+    </tr>
+    <tr>
+      <th>Tanzania</th>
+      <td>1794</td>
+    </tr>
+    <tr>
+      <th>Mozambique</th>
+      <td>1168</td>
+    </tr>
+    <tr>
+      <th>Senegal</th>
+      <td>1131</td>
+    </tr>
+    <tr>
+      <th>Gabon</th>
+      <td>1116</td>
+    </tr>
+    <tr>
+      <th>Cameroon</th>
+      <td>1114</td>
+    </tr>
+    <tr>
+      <th>Liberia</th>
+      <td>1065</td>
+    </tr>
+    <tr>
+      <th>Central African Republic</th>
+      <td>997</td>
+    </tr>
+    <tr>
+      <th>Nigeria</th>
+      <td>889</td>
+    </tr>
+    <tr>
+      <th>Congo Republic</th>
+      <td>678</td>
+    </tr>
+    <tr>
+      <th>Equatorial Guinea</th>
+      <td>38</td>
+    </tr>
+  </tbody>
+</table>
+
+
+The training and holdout sets contain examples of 30 animal species, plus some videos showing humans and
+a substantial number of blank videos.
+The following figure shows the number of videos containing each species.
+
+<img src="https://s3.amazonaws.com/drivendata-public-assets/zamba/td_full_set_number_videos_by_species.png" alt="" style="width:800px;"/>
```

Comment:
i'd much prefer committing the images to the repo instead of having them be in the public assets bucket. let me know what issue you ran into with including the relative links and i can help

Reply:
The relative paths that worked locally did not work on the site. I looked at similar pages and saw that they are pulling images from the public assets bucket, so I switched to that. It works both locally and on the site.